openapi/code_samples/PHP-SDK-3.0/api-keys/post.php

```diff
@@ -0,0 +1,10 @@
+$service = new Rebilly\Sdk\UsersService($client);
+
+$apiKeyForm = new Rebilly\Sdk\Model\ApiKey();
+$apiKeyForm->setDescription('Test key');
+
+try {
+    $apiKey = $service->apiKeys()->create($apiKeyForm);
+} catch (Rebilly\Sdk\Exception\DataValidationException $e) {
```

Comment:
If somebody were to copy the snippet over, this would be invalid as PHP resolves it as relative namespace. Better to include a leading backslash here an after.
```suggestion
} catch (\Rebilly\Sdk\Exception\DataValidationException $e) {
```

Reply:
Included the backslash for all the code samples.